cmd/release-controller-api/http_helper.go

```diff
@@ -240,12 +240,18 @@ func canLink(tag imagev1.TagReference) bool {
 	}
 }
 
-func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference) string {
+func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference, inconsistencies bool) string {
 	if canLink(tag) {
+
 		if value, ok := tag.Annotations[releasecontroller.ReleaseAnnotationKeep]; ok {
 			return fmt.Sprintf(`<td class="text-monospace"><a title="%s" class="%s" href="/releasestream/%s/release/%s">%s <span>*</span></a></td>`, template.HTMLEscapeString(value), phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name))
 		}
-		return fmt.Sprintf(`<td class="text-monospace"><a class="%s" href="/releasestream/%s/release/%s">%s</a></td>`, phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name))
+		if strings.Contains(tag.Name, "night") && inconsistencies {
```

Comment:
I would match on `nightly` here, not `night`

Reply:
Fixed.